app/models/dwelling.rb

```diff
@@ -12,19 +12,27 @@ class Dwelling < ApplicationRecord
 
   attr_accessor :audit_planning_application_id
 
-  TENURES = %w[open social intermediate].freeze
+  TENURES = %w[open social intermediate other].freeze
   TENURE_PRODUCTS = [
```

Comment:
Is this a sensible order? Is there an invisible structure here?
I'd vaguely expect `London Living Rent` and `London Living Rent Equivalent` to be adjacent.
Alphabetical probably isn't awful; I don't know if there's any better way.

Reply:
Yeah, I don't think there's a particular order to them, I thought I'd order them in the same way they were in the table in the ticket.